numba_cuda/numba/cuda/lowering.py

```diff
@@ -31,7 +32,11 @@
 from numba.core.environment import Environment
 from numba.core.analysis import compute_use_defs, must_use_alloca
 from numba.misc.firstlinefinder import get_func_body_first_lineno
-from numba.misc.coverage_support import get_registered_loc_notify
+
+numba_version = get_versions()["version"].split(".")
+numba_minor_version = int(numba_version[1])
```

Comment:
For future reference, we can get the version info with `numba.version_info.major`, `numba.version_info.minor`, etc.